File: 01-Basics/03-Count-Even-Odd/count_even_odd.py
arr = [4, 7, 2, 9, 6, 3]

even = 0
odd = 0

for i in arr:
    if i % 2 == 0:
        even += 1
    else:
        odd += 1

print("Even:", even)
print("Odd:", odd)

File: 01-Basics/03-Count-Even-Odd/CountEvenOdd.java
public class CountEvenOdd {
    public static void main(String[] args) {
        int[] arr = {4, 7, 2, 9, 6, 3};

        int even = 0;
        int odd = 0;

        for (int i = 0; i < arr.length; i++) {
            if (arr[i] % 2 == 0) {
                even++;
            } else {
                odd++;
            }
        }

        System.out.println("Even: " + even);
        System.out.println("Odd: " + odd);
    }
}